src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../Types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = (props) => (
+  <ul className="movies-list">
+    {props.movies.map((movie) => (
+      <li className="card" key={movie.imdbId}>
+        <MovieCard movie={movie} />
```

Comment:
```suggestion
        <MovieCard { ...movie } />
```

so you can destructure on MovieCard needed properties